swaglyrics/tab.py

```diff
@@ -1,21 +1,42 @@
 from swaglyrics.cli import lyrics
-from flask import Flask, render_template
+from flask import Flask, render_template,jsonify,request
+from flask_cors import CORS, cross_origin
 import os
 from swaglyrics import spotify
 
+
 app = Flask(__name__, template_folder=os.path.join(os.path.dirname(os.path.abspath('SwagLyrics-For-Spotify')), 'swaglyrics/templates'))
-# use relative path of the template folder
```

Comment:
why did you remove this comment?

Reply:
I had many different comments as well. So removed all comments for time being